README.md

```diff
@@ -54,24 +54,29 @@ This font can only be changed for the `Markdown viewer` and the `Terminal`. You
 
 ## 📦 Requirements
 
-- [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) >= 3.0
+- [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) < 4.0
```

Comment:
Oops meant to put the comment on this side. 

>= 3.0, <4.0

Reply:
Yes good catch